pkgs/development/python-modules/hdbscan/default.nix

```diff
@@ -0,0 +1,38 @@
+
+{ lib
+, buildPythonPackage
+, cython
+, numpy
+, scipy
+, scikitlearn
+, nose
+, pythonOlder
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "hdbscan";
+  version = "0.8.12";
+
+  src = fetchFromGitHub {
+    owner = "scikit-learn-contrib";
+    repo = "hdbscan";
+    rev = "51fd5ed225d971daa7658d94814c8237a75a03e4";
```

Comment:
That's not the revision for tag 0.8.12.

Reply:
Hmm I have always been taking the latest revision instead of the tags; I'll keep an eye for it now. Should be fixed now.